### DISCUSSION TITLE
RX4R or similar via D16 protocol?

### DISCUSSION THREAD

**Discussion by cru774** [pascallanger/DIY-Multiprotocol-TX-Module#665] [General]
Is an RX4R in D16 mode compatible with any of the listed RF modules? 
In other words: does the fact that the X4R is listed as compatible (FRSKYX - 15), and the RX4R is not, imply that it the RX4R is unsupported?

Thanks a bunch
Christian

https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md

**Comment by HazzaHFPV** [comment]
Yes, the RX4R can be used with the 4in1 module. Depending on the flashed firmware: FrSkyX or FrSkyX2 protocol

**Reply by cru774** [reply]
Brilliant, thanks for the quick confirmation.

**Reply by cru774** [reply]
Tx arrived, different Rx though: G-RX6. Failing to bind, trying the following:

- Power off Tx Rx
- Power on Tx, selecting Setup -> Internal RF, Multi, Frsky X, D16, Bnd, Ch1-8 telemetry on
- Power on Rx with bind button pressed

Rx green LED  flashes once, red LED still blinking. No bind.

Status: V1.3.2.77 AETR, latest OpenTx. Rx firmware Frsky ACCST D16 v2.1.2.

Wrong Rx? Also tested FrSkyX2 protocol and all Subtypes.

Thanks

**Reply by HazzaHFPV** [reply]
If the receiver is running accst 2.1.2, then FrSkyX2 D16 should be the protocol to use in the radio. Use FrSkyX for accst v1.x.x

Try putting the Rx into bind first. Both LEDs should be solid to indicate it is in bind mode. Then start the binding procedure in your radio as you did. The Rx will indicate a successful bind with solid green and blinking red LED. Thereafter re-power the Rx and the commands from the radio should be correctly received.

**Reply by cru774** [reply]
Genius :) FrSkyX2 LBT(EU) worked, D16 did not. Flashed to V1.3.3.7 also, that may have helped also. Really appreciate your help!
Thanks again
Christian

**Comment by HazzaHFPV** [comment]
Ah yes. Should have mentioned that if accst D16 LBT is flashed to the receiver that EU(LBT) needs to be selected in the radio.
But glad that you got it working :)
